<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-24T10:46:53 IST" name="Default suite" finished-at="2024-05-24T10:47:12 IST" duration-ms="18547">
    <groups>
    </groups>
    <test started-at="2024-05-24T10:46:53 IST" name="Default test" finished-at="2024-05-24T10:47:12 IST" duration-ms="18547">
      <class name="com.orangeHMR.TestCases.TestCase1">
        <test-method is-config="true" signature="Basic()[pri:0, instance:com.orangeHMR.TestCases.TestCase1@74e52303]" started-at="2024-05-24T10:46:53 IST" name="Basic" finished-at="2024-05-24T10:47:01 IST" duration-ms="7894" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Basic -->
        <test-method is-config="true" signature="Login()[pri:0, instance:com.orangeHMR.TestCases.TestCase1@74e52303]" started-at="2024-05-24T10:47:01 IST" name="Login" finished-at="2024-05-24T10:47:04 IST" duration-ms="2460" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method signature="Testcase01()[pri:0, instance:com.orangeHMR.TestCases.TestCase1@74e52303]" started-at="2024-05-24T10:47:04 IST" name="Testcase01" finished-at="2024-05-24T10:47:09 IST" duration-ms="5793" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.orangeHMR.TestCases.TestCase1.Testcase01(TestCase1.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Testcase01 -->
        <test-method is-config="true" signature="logout(org.testng.ITestResult)[pri:0, instance:com.orangeHMR.TestCases.TestCase1@74e52303]" started-at="2024-05-24T10:47:09 IST" name="logout" finished-at="2024-05-24T10:47:11 IST" duration-ms="1928" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Testcase01 status=FAILURE method=TestCase1.Testcase01()[pri:0, instance:com.orangeHMR.TestCases.TestCase1@74e52303] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.orangeHMR.TestCases.TestCase1@74e52303]" started-at="2024-05-24T10:47:11 IST" name="close" finished-at="2024-05-24T10:47:12 IST" duration-ms="399" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.orangeHMR.TestCases.TestCase1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
